webapp/packages/plugin-authentication/src/Dialog/AuthDialog.tsx

```diff
@@ -144,6 +147,7 @@ export const AuthDialog: DialogComponent<IAuthOptions, null> = observer(function
       currentTabId={state.tabId}
       onChange={tabData => {
         state.setTabId(tabData.tabId);
+        state.resetErrorState();
```

Comment:
Probably it's more convenient to call resetErrorState in setTabId (setTabId can be renamed to switchAuthMode/setAuthProvider)